libs/ledger-live-common/src/mock/fixtures/cryptoCurrencies.ts

```diff
@@ -0,0 +1,35 @@
+import type { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+
+export default function createCryptoCurrency(family: string): CryptoCurrency {
```

Comment:
Nitpicking here but could you please name this `createMockCryptoCurrency`?
Ideally we'd like to have a safety mechanism that throws an error when this is used outside of test environment, but the naming should be enough to avoid mistakes.

Reply:
I agree to avoid any confusion.
But I would prefer to find some different name as this is not a mock. This is a testing data (or fixture). A mock is an dummy object (or function) that respond something consistent to its caller (but without any true logic inside).